cudax/include/cuda/experimental/__async/sender/conditional.cuh

```diff
@@ -60,9 +59,9 @@ struct __cond_t
   template <class _Pred, class _Then, class _Else>
   struct __data
```

Comment:
Its a bit inconsistent to have a private struct with public names. Should we also make the struct available?